### PR TITLE
V1.20.0

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

Changelog - 
- New: Lottie Block.
- Improvement: Post Layout: Added EditMode to shuffle the Post elements i.e Post Title, Post Meta, Post Content, Featured Image, CTA Button etc.
- Improvement: Better options for delete/edit/duplicate action of each content in Content timeline blocks.
- Fix: Price List - CSS issue on mobile.
- Fix: Testimonial - Fixed the JS console warning with Slick library.
- Fix: Taxonomy List - Fixed the JS console warning in editor mode.
- Fix: Table of content - Fixed the pipe symbol issue in heading.
- Fix: Price List - Fixed missing image padding option and columns not updating issue.
- Fix: Gravity Form Styler - Multistep form styling issue.
- Fix: FAQ Schema - Fixed Accessibility issue.
- Fix: FAQ Schema - Fixed child block not inheriting question tag from the parent block when adding a new question.
- Fix: FAQ-Schema - FAQ console error when accessing child by hashval.
- Fix: Taxonomy List - Fixed issue with broken links.
- Fix: Table of Content - IE11 compatibility issues.
- Fix: Testimonial - Fixed carousel not being displayed on RTL websites.
- Fix: Post Layout - Child Posts not visible when parent category is selected in editor mode.

### Types of changes

- V1.20.0

### How has this been tested?

- Tested by QA Team.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by amithire1201 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

File: readme.txt [hunk 1]
```diff
@@ -155,12 +156,34 @@ Absolutely no! The Ultimate Addons for Gutenberg is built with ease and performa
...
+* Improvement: Better options for delete/edit/duplicate action of each content in Content timeline blocks.
+* Fix: Price List - CSS issue on mobile.
+* Fix: Testimonial - Fixed the JS console warning with Slick library.
+* Fix: Taxonomy List - Fixed the JS console warning in editor mode.
+* Fix: Table of content - Fixed the pipe symbol issue in heading.
+* Fix: Price List - Fixed missing image padding option and columns not updating issue.
+* Fix: Gravity Form Styler - Multistep form styling issue.
+* Fix: FAQ Schema - Fixed Accessibility issue.
+* Fix: FAQ Schema - Fixed child block not inheriting question tag from the parent block when adding a new question.
+* Fix: FAQ-Schema - FAQ console error when accessing child by hashval.
```
Comment (vrundakansara) [inline readme.txt:181]: `* Fix: FAQ-Schema - FAQ console error when accessing child by link hash value.`
Let's change it to this.

**Review by mangeshsonule — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 1 above)
Comment (AbhijitNage123) [reply readme.txt]: Done

File: README.md [hunk 2]
```diff
@@ -155,12 +156,34 @@ Absolutely no! The Ultimate Addons for Gutenberg is built with ease and performa
...
+
+The awesome React FontIconPicker is developed by [Alessandro Benoit](http://codeb.it/) & [Swashata](https://github.com/swashata). Our external packages uses [Rating Star Component](https://github.com/n49/react-stars) that are distributed under the terms of the ISC. While [Slick](https://github.com/akiran/react-slick), [Lottie](https://github.com/chenqingspring/react-lottie) and [Striptags](https://github.com/ericnorris/striptags) is distributed under the terms of the MIT.
+
 ## Screenshots ##
 
 1. /assets/screenshots/1.png
 
 ## Changelog ##
 
+### 1.20.0 ###
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply README.md:169]: Done

**Review by supriyajjadhav — APPROVED** [review]

File: src/blocks/post/post-title/save.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+export default function save() {
+    return <h3 className="uagb-post__text uagb-post__title">
+                <a href='#' target='_blank' rel ="noopener noreferrer">Wordpress Post Title</a>
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply src/blocks/post/post-title/save.js]: Done

File: src/blocks/post/post-meta/save.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+
+export default function save() {
+    return <div className='uagb-post-grid-byline'>
+            <span className='uagb-post__author'>
+                <span className="dashicons-admin-users dashicons"></span>
+                Author Name
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply src/blocks/post/post-meta/save.js]: Done

File: src/blocks/post/post-excerpt/save.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+export default function save() {
+    return <div className='uagb-post__excerpt uagb-full_post'>
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply src/blocks/post/post-excerpt/save.js]: Done

File: src/blocks/post/post-image/save.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+export default function save() {
+    return <div className='uagb-post__image'>
+                <img
+                    src='https://www.ultimategutenberg.com/wp-content/uploads/2018/12/uagb-10.png'
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply src/blocks/post/post-image/save.js]: Done

**Review by mangeshsonule — APPROVED** [review]
PR Good to me

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**